styles.css

```diff
@@ -0,0 +1,30 @@
+navbar{
+    position: top;
+
+}
+header
+{
+    position: fixed;
+    width: 100%;
+}
+
+@media (max-width: 650px){
+    #ul{
+        flex-direction: column;
+
+    }
+}
+header {
+    position: fixed;
+    top 0;
+    min-height: 15px;
+    padding: 0px 20px;
+    display: flex;
+    justify-content: space-around;
+    align-items: center;
+    background-color; #ab8103
+    @media(max-width: 600px)
+    {
+        flex-wrap: wrap;
+    }
+}
```

Comment:
Also the css file is still incomplete

Reply:
[ ] kindly change this one 🤓👀 , the comments should sound more like suggestions instead of orders. For example:
_"I think it would be better to use semantic tags"_